test/multiple-element-interactions-test.js

```diff
@@ -42,53 +44,55 @@ describe("When multiple DOM elements are present", () => {
             });
         });
 
-        it("can read attributes from custom child element's prototypes", () => {
-            var DataSource = components.newElement();
-            DataSource.data = [1, 2, 3];
-            components.registerElement("data-source", { prototype: DataSource });
+        // Pending until we decide on a good solution
```

Comment:
Eventually yes, but not for now, no. We just need the test to not be disabled (xit -> it). We can make the 'how do you talk to child elements' nicer separately later on.